src/components/Header.tsx

```diff
@@ -1,3 +1,30 @@
-export const Header = () => {
-	return <h1>TO-DO-LIST</h1>;
+import "./header.css";
+import { Login } from "./Login";
+import { SignIn } from "./SignIn";
+
+interface propsH {
+	header: string;
+	headerParagraph: string;
+}
+
+export const Header = ({ header, headerParagraph }: propsH) => {
+	return (
+		<div className="flex items-center ">
+			<div className="ml-10 ">
+				<h1 className="text-6xl mt-4 ">{header}</h1>
+				<p className="leading-9 mt-4 text-slate-600 ">{headerParagraph}</p>
+				<div className="flex mt-4">
+					<Login login="Login" />
+					<SignIn signin="Sign in" />
+				</div>
+			</div>
+			<div>
+				<img
+					className="header__img mr-4  block"
+					src="../src/assets/dog.png"
```

Comment:
v headeru bude vzdy tahle jedna fotka? Pokud ano tak v dobry, akorat to naimportuj nahre a sem to vloz jako promennou. pokud ne, tak to sem proval pres props

Reply:
Jinak si rozklikni preview tehle brancha a koukni jak vypada logo